### PR TITLE
Add easyStart function

### PR DESCRIPTION
**Pull request by teriaiw**

### CODE REVIEW

File: src/lib/transformer.ts [hunk 1]
```diff
@@ -59,4 +60,24 @@ export class Transformer {
   static reset() {
     this.basisMatrix = math.matrix([[1, 0, 0], [0, 1, 0], [0, 0, 1]]);
   }
+
+  static easyStart() {
```
Comment (TamirShklaz) [inline src/lib/transformer.ts:64]: This function should not be in the Transformer class. This is a utility function that uses the transformer class. So make a new file called utils.ts and put this function in there.

File: src/lib/transformer.ts [hunk 2]
```diff
@@ -59,4 +60,24 @@ export class Transformer {
...
+    background("black");
+    translate(0, height)
```
Comment (TamirShklaz) [inline src/lib/transformer.ts:66]: Use Strive.translate, Strive.rotate, Strive.scale for all transformations

File: src/lib/transformer.ts [hunk 3]
```diff
@@ -59,4 +60,24 @@ export class Transformer {
...
+    this.scale(1,-1)
+    stroke("white")
+    fill("white")
+    textSize(18)
+    tickAxes()
+    stroke("green")
+    strokeWeight(1)
+    let y = height-mouseY
+    line(0, y, mouseX, y)
```
Comment (TamirShklaz) [inline src/lib/transformer.ts:75]: You can get the transformed mouseX and mouseY from Transformer.mouse if you use Strive.translate, Strive.scale, Strive.rotate for all transformations

**Review by TamirShklaz — CHANGES_REQUESTED** [review]

**Comment by teriaiw** [comment]
New branch created to follow git flow

**Pull request closed without merging**